cmd/partitiontest_linter/analyzer/analyzer_test.go

```diff
@@ -0,0 +1,12 @@
+package analyzer_test
```

Comment:
why not `analyzer`? it is a unit test for a sibling file. By doing this 1) is it the way 2) no analyzer import needed below

Reply:
this one also fails for me:
```
Running tool: /usr/local/go/bin/go test -timeout 30s -coverprofile=/var/folders/m7/v0208hbj5p7643101fcjkpl00000gp/T/vscode-go78tJNa/go-code-cover github.com/algorand/go-algorand/cmd/partitiontest_linter/analyzer

# github.com/algorand/go-algorand/cmd/partitiontest_linter/analyzer [github.com/algorand/go-algorand/cmd/partitiontest_linter/analyzer.test]
/Users/jack/projects/algojack.go-algorand/cmd/partitiontest_linter/analyzer/analyzer_test.go:11:47: undefined: analyzer
FAIL	github.com/algorand/go-algorand/cmd/partitiontest_linter/analyzer [build failed]
FAIL
```

Reply:
if `analyzer.go` and `analyzer_test.go` are in the same package, `Analyzer` var does not need package qualifier.

Reply:
fixed, thank you